.travis.yml

```diff
@@ -165,33 +165,24 @@ script:
     - if [ -n "$DESTDIR" ]; then
           cd _build;
       fi
-    - if $make update; then
-          echo -e '+\057 MAKE UPDATE OK';
-      else
-          echo -e '+\057 MAKE UPDATE FAILED';
+    - if ! $make update; then
+          echo -e '\052\052 FAILED -- MAKE UPDATE';
           travis_terminate 1;
       fi
-    - git diff --exit-code
-    - if [ -n "$CHECKDOCS" ]; then
-          if $make doc-nits; then
-              echo -e '+\057\057 MAKE DOC-NITS OK';
-          else
-              echo -e '+\057\057 MAKE DOC-NITS FAILED';
-              travis_terminate 1;
-          fi;
+    - if ! git diff --exit-code; then
+          echo -e '\052\052 FAILED -- UPDATED FILES NOT COMMITED';
+          travis_terminate 1;
       fi
-    - if [ -n "$GENERATE" ]; then
-          if $make build_all_generated; then
-              echo -e '+\057\057\057 MAKE BUILD_ALL_GENERATED OK';
-          else
-              echo -e '+\057\057\057 MAKE BUILD_ALL_GENERATED FAILED';
-              travis_terminate 1;
-          fi;
+    - if test -n "$CHECKDOCS" -a ! $make doc-nits; then
```

Comment:
That doesn't look right. Shouldn't that be `if test -n "$CHECKDOCS" && ! $make doc-nits; then` ?